ViewGenerator/tools/webpack/Plugins/Utils.ts

```diff
@@ -83,39 +88,6 @@ export function generateManifest(
     return entryArrayManifest;
 }
 
-/*
- * Custom typescript error formater for Visual Studio.
- * */
-export function customErrorFormatter(issue: Issue, enableColors: boolean, namespace: string) {
```

Comment:
we still have people using the VS, does removing this section impact them?

Reply:
I think it's not working properly as of today. We can reiterate if needed, this just has visual impacts

Reply:
ok, fair point